install-and-configure/advanced-configuration/gpu.md

```diff
@@ -0,0 +1,55 @@
+# NVIDIA GPU Monitoring Configurations
+
+## Monitoring GPU utilization
+
+In order for Kubecost to understand a container's GPU utilization percentage, Kubecost depends on metrics being available from NVIDIA [DCGM Exporter](https://github.com/NVIDIA/dcgm-exporter). Follow the below instructions to install and configure DCGM Exporter on each of your GPU-enabled clusters.
+
+```sh
```

Comment:
Need to add a paragraph here explaining that DCGM Exporter Pods must only run on nodes with NVIDIA GPUs. The below Helm command installs with no sort of Node affinity/selection, which is incorrect. General guidance should be:

1. If your GPU Nodes already have a label you wish to use, use these values <insert values snippet>.
2. If your GPU Nodes either do not have any indicative labels or you are not sure, we recommend installed GPU Feature Discovery (GFD) along with Node Feature Discovery (NFD). Here is the command to do so <insert instructions>. Once installed, here are the Helm values to deploy DCGM Exporter with Node affinity set to use labels created automatically by NFD <insert values snippet>.